assets/src/edit-story/components/elementBorder/index.js

```diff
@@ -54,6 +63,16 @@ export default function WithBorder({ element, previewMode = false, children }) {
     dataToEditorX: state.actions.dataToEditorX,
     dataToEditorY: state.actions.dataToEditorY,
   }));
+  const ref = useRef(null);
+  const { id } = element;
+
+  useCSSVarColorTransformHandler({
+    id,
+    targetRef: ref,
+    expectedStyle: 'border-color',
+    cssVar: '--element-border-color',
+  });
+
```

Comment:
This works great for a center border for a `MediaEdit` component. But an outside border for `MediaEdit` seems broken - it's completely missing? I know it's not directly related to this of course.

Reply:
Hmm, we should probably create a separate issue for the media edit + outside border -- media edit mode is cutting off the overflow outside of the visible area (relevant mainly in case of zoomed media) so I don't currently even have a good idea how to add the outside border for media edit. It would need a be a separate element added specifically for that.

Added new issue for it: https://github.com/google/web-stories-wp/issues/5440